HedvigInsurance/racoon
# GRW-2349 / feature / Cart Product Recommendation Cross Sell 🛒

## Describe your changes

Added offers to the cart page when the user has home insurance in their cart.

**Test workflow**
- add home to your cart
- Go to cart
- should see accident offer
- add accident offer
- remove accident offer
## [Figma](https://www.figma.com/file/8SfzuQEJ4LJ6uKL0l7WKoE/Hedvig.com-%C2%B7-Final-Design?node-id=6424%3A74806&t=RdHy8Uk5AK2Chp9e-0)

<img width="642" alt="Screenshot 2023-03-08 at 16 31 50" src="https://user-images.githubusercontent.com/50870173/223756441-fa620a0f-504a-4864-a842-c0bff9e3e0bf.png">

## Video
https://user-images.githubusercontent.com/50870173/223751768-0a91c2d3-323f-4e34-9ee1-297232cf9392.mov

## Jira issue(s): [GRW-2349]

## Justify why they are needed

- quickly allow users to also buy accident insurance
- 
## Checklist before requesting a review



[GRW-2349]: https://hedvig.atlassian.net/browse/GRW-2349?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
